InfiniteLoopers-App/src/main/java/com/isa/account/UserManager.java

```diff
@@ -7,11 +7,12 @@
 import java.io.IOException;
 import java.util.ArrayList;
 import java.util.List;
+import java.util.Scanner;
 
 public class UserManager {
     private final ObjectMapper objectMapper = new ObjectMapper();
     private final UserList userList = new UserList();
-    private final String databasePath = "users.json";
+    private final String databasePath = "src/main/resources/users.json";
```

Comment:
zobacz czy napewno dziala ten path bo ja mialem tez duzo problemow z tymi .jsonem i czasem sie zapisywalo czasem nie. Z Mackiem chcielismy to ogarnac i raz bylo dobrze raz nie.